### DISCUSSION TITLE
[FR] Larger Sized Cards

### DISCUSSION THREAD

**Discussion by oerix** [Clooos/Bubble-Card#932] [Feature requests]
Any ideas when we can expect Bigger cards to be implemented? 

At the moment, all the cards look the same. By using different sized cards, there are so many possibilities (media player, climate etc.) which are much more touch friendly just like mushroom cards.

Thanks

**Comment by Clooos** [comment]
Hi! This should come in a few releases, this is in my priorities 🙂

**Reply by oerix** [reply]
Amazing, thank you for all your work!

**Comment by Callidus2000** [comment]
Hi,
looking forward to this feature.
I'm using the bubble-card as default for most of my dashboards. On a larger wallpanel it would be nice to be able to make the buttons huge. Important: without vast CSS voodoo, maybe by using an internal zoom attribute (which is used to calculate all width/heights etc).

![image](https://github.com/user-attachments/assets/b530d052-68b4-47c5-abfc-de61b87b2f59)

**Comment by Clooos** [comment]
Hi! By the way, this feature is now part of Bubble Card v3.1.0-beta, here's the changelog 😄

https://github.com/Clooos/Bubble-Card/releases/tag/v3.1.0-beta.1
